enterprise/developerbox/build/README.md

```diff
@@ -16,6 +20,47 @@ permalink: /documentation/enterprise/developerbox/build/
 
 ***
 
+# Build SCP Firmware From Source
+
+Developerbox contains SCP(System Control Processor) of ARM@Cortex-M3(CM3).
+It manages power domain and enables application core boot.
```

Comment:
This sentence is perfectly correct but it is not idiomatic English. It would be more readable for native speakers with different phrasing. Perhaps replace with the following:

Developerbox contains a Cortex-M3 System Controller Processor (SCP) which manages system power and is responsible for booting the main processor.

Reply:
Thank you for the comments. I will update patch to follow all your comments.